pkg/virt-launcher/virtwrap/device/hostdevice/hostdev.go

```diff
@@ -141,9 +141,10 @@ func createMDEVHostDevice(hostDeviceData HostDeviceMetaData, mdevUUID string) (*
 				UUID: mdevUUID,
 			},
 		},
-		Type:  api.HostDeviceMDev,
-		Mode:  "subsystem",
-		Model: "vfio-pci",
+		Managed: "no",
```

Comment:
What is the issue with omitting `Managed` attribute? It should be fine according to the libvirt doc, no?

https://libvirt.org/formatdomain.html#host-device-assignment

> Note: The managed attribute is only used with type='pci' and is ignored by all the other device types, thus setting managed explicitly with other than a PCI device has the same effect as omitting it.

Reply:
Yeah, the functionality doesn't change as far as I could see but still, Libvirt does a validation and produces an error when we leave this empty (also it will default it to "no" afterward). As an alternative, I can make the `Managed` optional so it will not render in the xml, which when I think about it would be what we want. WDYT?

Reply:
Ah, so it is not marked as `omitempty` in 

https://github.com/kubevirt/kubevirt/blob/ac71a19578ac07a8b3a07dcbfbaf0cf253a04bf2/pkg/virt-launcher/virtwrap/api/schema.go#L568

and therefore it is rendered as an empty string `""`, right?

If so, then I would rather make the field optional in the schema. I think that way it will closer follow the libvirt spec, and we will not need to add it where it is in fact not required (e.g. here for mdev).